myapp/spec/factories/constant.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :constant do
+    key { 'maintenance_mode' }
+    value { 'true' }
+  end
```

Comment:
おそらく設計思想としてはメンテナンス専用のテーブルではなさそうなので、`constat` と名前を指定してできるレコードがデフォルトでメンテナンス用になっていると、後に別の用途が増えたときに使いにくい感じがするので、デフォルト値はないほうが良さそうに思います。
現在もSpec中でキーも含めて指定しているし。
```suggestion
  factory :constant do
  end
```

Reply:
`constant`というテーブル名にした上に、デフォルトでメンテナンスになっているのは良くないですね。。
確かにデフォルト値はない方が良いと思いました！修正しました！
https://github.com/Fablic/training/pull/1352/commits/2949f9ae4ebc6796b7ce7a8b9f51c0e8f83d4bda